viz.yaml

```diff
@@ -232,13 +232,22 @@ visualize:
     title: "SVG skeleton"
     alttext: "Beginning of the full storm SVG"
 publish:
+  -
+    mimetype: application/javascript
```

Comment:
what's this line for?

Reply:
whoops - forgot to delete it when I deleted the d3 library load